haiku-apps/beshare/beshare-2.31.recipe

```diff
@@ -13,16 +13,19 @@ have to click \"refresh\" or anything first)
 - BeShare supports any number of simultaneous uploads and downloads \
 in parallel, but will serialize simultaneous file requests to any \
 given host for efficiency."
+HOMEPAGE="https://github.com/HaikuArchives/BeShare"
 COPYRIGHT="1999-2012 Jeremy Friesner"
 LICENSE="
 	Public Domain
 	libsanta
 	"
-HOMEPAGE="https://github.com/HaikuArchives/BeShare"
-SOURCE_URI="$(HOMEPAGE)/archive/b721fcc.tar.gz"
+SOURCE_URI="${HOMEPAGE}/archive/b721fcc.tar.gz"
```

Comment:
Wouldn't it be slightly better to use:
```
srcGitRev="b721fcc4a4297b32738994b9c427d7db26c2c8fc"
SOURCE_URI="$HOMEPAGE/archive/$srcGirRev.tar.gz"
```
BTW, the latest revision is:
```
srcGitRev="2345e48664fddfff1d9a968d1da2a185dbbdb50c"
```
but switching to that would require updating CHECKSUM_SHA256SUM and testing the build.